release/test_release_gold.sh

```diff
@@ -7,7 +7,14 @@ source release_gold.sh --test
 function main {
 	set_up
 
+	if [ "${?}" -ne 0 ]
+	then
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
```

Comment:
@kiwm, is it really necessary? Which function will be consuming the return of this function?